rust/Cargo.toml

```diff
@@ -40,3 +40,4 @@ arrow-array = { version = "53.1.0", default-features = false, features = [
 arrow-buffer = { version = "53.1.0", default-features = false }
 arrow-schema = { version = "53.1.0", default-features = false }
 arrow-select = { version = "53.1.0", default-features = false }
+arrow-cast = { version = "53.1.0", default-features = false }
```

Comment:
Looking at this again, I think we can remove it because it is only used in the `README`.

```suggestion
```

Reply:
done